### PR TITLE
Display something on the browser

### PR DESCRIPTION
**Pull request by 552020**
- Lil modification on the mini_websserv in my repo, which displays some html. 
- Usage is explained in the README
- Some docs have been added
- Also some stuff in the .vscode directory: this are the settings I have in my cpp projects, if you have other settings we can agree on it, or not commit the .vscode dir at all (which was already committed)
- I add also a .clang_format file, for the formatting of the code: here is important the we find a common formatting style so that we don't get modifications in git cause of formatting stuff, which makes code reviews much harder.

### CODE REVIEW

**Comment by dantol29** [comment]
Great documentation! Man.cpp totally makes sense!

File: .vscode/c_cpp_propertied.json
Comment (lmangall) [inline .vscode/c_cpp_propertied.json:1]: Can you explain how this json is being used ? especially :
      "cppStandard": "c++98",

**Review by lmangall — APPROVED** [review]
Thanks for the documentation. I heard of Postman and it seems to be a tool one has to master, have you ever used it ?

File: .vscode/c_cpp_propertied.json
Comment (552020) [reply .vscode/c_cpp_propertied.json:1]: I guess it tells to the language server of VSCode that the C++ version being used is C++98, so it should underline in red if you use features which are not part of C++98 (like auto). But it never happend to me that I got this warning.

**Comment by 552020** [comment]
> Great documentation! Man.cpp totally makes sense!

Thanks! @dantol29! What about approving it? :-)

**Comment by 552020** [comment]
> Thanks for the documentation. I heard of Postman and it seems to be a tool one has to master, have you ever used it ?

@lmangall I did in many Udemy courses when you need to test your application. It's pretty easy to use. You can download it and make the same request you did through the browser to our server with postman: you just put the same address and you will see the response. You can get the same information also in the dev tool of the browser (like the time of the response and the content of the header and body of the http request). But in postman you can save different requests and you can perform them programmatically.

**Review by dantol29 — APPROVED** [review]
Great!

**Pull request merged**